src/components/navBar/navBar.js

```diff
@@ -434,6 +439,21 @@ function MdNavItem($$rAF) {
           scope.$apply();
         });
       });
+
+      var stopDisabledWatch = angular.noop;
+
+      if (angular.isDefined(attrs.disabled)) {
+        mdNavItem.disabled = true;
+      }
+      else if (attrs.ngDisabled) {
+        stopDisabledWatch = scope.$parent.$watch(attrs.ngDisabled, function() {
+          mdNavItem.disabled = $parse(attrs.ngDisabled)(scope.$parent);
+        });
+      }
```

Comment:
I didn't use $observe because it adds another watcher, but now that I think about it I can remove the ngDisabled watcher so it doesn't matter anyway.